src/Workspaces/Core/Portable/Remote/WellKnownRemoteHostServices.cs

```diff
@@ -6,7 +6,7 @@ internal class WellKnownRemoteHostServices
     {
         public const string RemoteHostService = "remoteHostService";
         public const string RemoteHostService_Connect = "Connect";
-        public const string RemoteHostService_SynchronizeAsync = "SynchronizeAsync";
```

Comment:
RIght now, someone could rename "SynchronizePrimaryWorkspaceAsync" and not know to update this location.  The nameof approach addresses that.